Corona-Warn-App/src/main/res/values-ar/strings.xml

```diff
@@ -469,6 +475,8 @@
     <!-- XACT: settings(tracing) - describes illustration -->
     <string name="settings_tracing_bluetooth_illustration_description_inactive">"أوقف شخص تشغيل Bluetooth على هاتفه الذكي؛ وبالتالي، لم يتم تسجيل تجمُّع مع شخصين آخرين."</string>
     <!-- XACT: settings(tracing) - describes illustration -->
+    <string name="settings_tracing_location_illustration_description_inactive">""</string>
+    <!-- XACT: settings(tracing) - describes illustration -->
     <string name="settings_tracing_connection_illustration_description_inactive">"قطع شخص الاتصال بالإنترنت على هاتفه الذكي؛ وبالتالي، لم يتم تسجيل تجمُّع مع شخصين آخرين."</string>
```

Comment:
Hi Oliver, can you remove all language specific changes (ar, en, tr, ru)? Only the german text is the values-de file is needed and the empty placeholder in the value-base file.